test/timezone_names.jl

```diff
@@ -0,0 +1,7 @@
+import TimeZones: timezone_names
+
+names = timezone_names()
+
+@test length(names) >= 429
```

Comment:
I specifically did `>=` to handle the creation of timezones. I don't expect that the total number of zones will decrease.